Nav/OnBoarding/Preferences.js

```diff
@@ -1,41 +1,38 @@
-import React, {Component} from 'react';
+import 'react-native-gesture-handler';
+import React, { Component } from 'react';
+import AppIntroSlider from 'react-native-app-intro-slider';
 import {
   SafeAreaView,
   StyleSheet,
   Text,
   View,
   TouchableOpacity,
   ScrollView,
+  Image,
 } from 'react-native';
-import {bindActionCreators} from 'redux';
-import {connect} from 'react-redux';
-import {updateFTUE} from '../actions';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { updateFTUE } from '../actions';
 import PropTypes from 'prop-types';
 import colors from '../../assets/colors';
 import Notification from '../../Settings/Notification';
 import Location from '../../Settings/Location';
 import Import from '../../Settings/Import';
 import AnalyticsOptIn from '../../Settings/AnalyticsOptIn';
-import {strings} from '../../locales/i18n';
+import { strings } from '../../locales/i18n';
+import { SetStoreData } from '../../utils/asyncStorage';
 
 class Preferences extends Component {
-  completeFTUE = () => {
-    this.props.updateFTUE({
-      field: 'enableFTUE',
-      value: 'false',
-    });
-  };
-
-  render() {
-    const {navigate} = this.props.navigation;
-
-    return (
-      <SafeAreaView style={styles.container}>
-        <ScrollView>
-          <View style={styles.intro_container}>
-            <Text style={styles.intro_text}>{strings('global.stopping')}</Text>
-          </View>
-          <View style={styles.settings}>
+  renderItem = ({ item }) => {
+    if (item.key === 'pref_5') {
+      return (
+        <SafeAreaView style={styles.intro_container}>
+          <View>
+            <View style={styles.content}>
+              <Image style={styles.icon} source={item.image}></Image>
+              <Text style={styles.content_title_text}>{item.title}</Text>
+            </View>
+            {/* <View style={styles.settings}>
```

Comment:
The functionality of the Preference page is broken and the style breaks. Let's break down this PR
1) sending the first PR to remove the privacy bar and update the privacy panel text.
2) Open a PR with separate commits for each panel notification/location/google import/analytics, we review by each commit.